neonvm/runner/main.go

```diff
@@ -480,6 +480,39 @@ func checkDevTun() bool {
 	return mode&os.ModeCharDevice == os.ModeCharDevice
 }
 
+func runInitScript(script string, logger *zap.Logger) error {
```

Comment:
```suggestion
func runInitScript(logger *zap.Logger, script string) error {
```

style nit: Generally logger, context, things like that are at the beginning of the args

Reply:
Fixed. 

In the Go teams I worked before it was usually:
 - context is the first argument
 - logger is the last argument

But I am happy with whatever as long as it is consistent.

Reply:
huh, interesting! yeah, in this codebase we have this pattern I guess :sweat_smile: